arangod/GeneralServer/VstCommTask.cpp

```diff
@@ -252,6 +252,13 @@ bool VstCommTask<T>::processMessage(velocypack::Buffer<uint8_t> buffer,
     }
     stat.SET_REQUEST_TYPE(req->requestType());
 
+    // process OPTIONS requests similar to how it is done for HTTP
+    if (req->requestType() == rest::RequestType::OPTIONS) {
```

Comment:
CORS requests are only used for http preflight requests

Reply:
I was told that VST should behave as close as possible to HTTP. For OPTIONS requests that was not the case so far. What would you suggest instead?

Reply:
not handling OPTIONS requests perhaps? https://livebook.manning.com/book/cors-in-action/chapter-4/

Reply:
it only makes sense in the context of a webbrowser

Reply:
Is there any harm in supporting this in VST, even though VST clients will make no use of it?
That way the differences in behavior would indeed be kept as small as possible. And IMHO it is unlikely that supporting OPTIONS requests in the server-side VST handler breaks anything.

Reply:
I agree, that it doesn't really make sense for VST. Until now the integration tests have issued an OPTIONS request as indicator for when the statistics updates have been processed - that is why I stumbled over this difference in the first place.   
However, I have now found a simple solution to enforce a sync of the statistics update as part of the request (as @jsteemann has preferred in the first place), so from that perspective this change would be obsolete now. It is up to you - should I remove this change or keep it?

Reply:
@mpoeter : ok, then let's remove OPTIONS handling from the VstCommTask for good.